python/lsst/eotest/sensor/BFTask.py

```diff
@@ -0,0 +1,271 @@
+"""
+brief Compute the brighter fatter correlation coefficients.  Input is
+flat pairs.  Based on the LSST DM cdoe.  Output is the nearest neighbor coefficients per amp.
+"""
+import os
+import glob
+import numpy as np
+
+import astropy.io.fits as fits
+from lsst.eotest.fitsTools import fitsTableFactory, fitsWriteto
+import lsst.eotest.image_utils as imutils
+from lsst.eotest.sensor.MaskedCCD import MaskedCCD
+
+import lsst.afw.geom as afwGeom
+import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
+
+import lsst.eotest.image_utils as im_util
+from lsst.eotest.sensor import MaskedCCD, AmplifierGeometry
```

Comment:
There are several imports that aren't being used anywhere.  From `pylint`:
```
BFTask.py:14:0: W0611: Unused lsst.afw.geom imported as afwGeom (unused-import)
BFTask.py:15:0: W0611: Unused lsst.afw.image imported as afwImage (unused-import)
BFTask.py:20:0: W0611: Unused lsst.eotest.image_utils imported as im_util (unused-import)
BFTask.py:21:0: W0611: Unused AmplifierGeometry imported from lsst.eotest.sensor (unused-import)
BFTask.py:22:0: W0611: Unused makeAmplifierGeometry imported from lsst.eotest.sensor.AmplifierGeometry (unused-import)
```
Could you remove these please?